cli/azd/pkg/infra/azure_resource_types.go

```diff
@@ -3,12 +3,51 @@ package infra
 type AzureResourceType string
 
 const (
-	AzureResourceTypeResourceGroup       AzureResourceType = "Microsoft.Resources/resourceGroups"
-	AzureResourceTypeDeployment          AzureResourceType = "Microsoft.Resources/deployments"
-	AzureResourceTypeStorageAccount      AzureResourceType = "Microsoft.Storage/storageAccounts"
-	AzureResourceTypeKeyVault            AzureResourceType = "Microsoft.KeyVault/vaults"
-	AzureResourceTypePortalDashboard     AzureResourceType = "Microsoft.Portal/dashboards"
-	AzureResourceTypeAppInsightComponent AzureResourceType = "Microsoft.Insights/components"
-	AzureResourceTypeWebSite             AzureResourceType = "Microsoft.Web/sites"
-	AzureResourceTypeContainerApp        AzureResourceType = "Microsoft.App/containerApps"
+	AzureResourceTypeResourceGroup         AzureResourceType = "Microsoft.Resources/resourceGroups"
+	AzureResourceTypeDeployment            AzureResourceType = "Microsoft.Resources/deployments"
+	AzureResourceTypeStorageAccount        AzureResourceType = "Microsoft.Storage/storageAccounts"
+	AzureResourceTypeKeyVault              AzureResourceType = "Microsoft.KeyVault/vaults"
+	AzureResourceTypePortalDashboard       AzureResourceType = "Microsoft.Portal/dashboards"
+	AzureResourceTypeAppInsightComponent   AzureResourceType = "Microsoft.Insights/components"
+	AzureResourceTypeLogAnalyticsWorkspace AzureResourceType = "Microsoft.OperationalInsights/workspaces"
+	AzureResourceTypeWebSite               AzureResourceType = "Microsoft.Web/sites"
+	AzureResourceTypeStaticWebSite         AzureResourceType = "Microsoft.Web/staticSites"
+	AzureResourceTypeServicePlan           AzureResourceType = "Microsoft.Web/serverfarms"
+	AzureResourceTypeSqlDatabase           AzureResourceType = "Microsoft.Sql/servers"
+	AzureResourceTypeCosmosDb              AzureResourceType = "Microsoft.DocumentDB/databaseAccounts"
```

Comment:
A few missing types we can add that we use in our samples
- Container App Environment
- Container App
- Cosmos Database
- Cosmos Database collection

Reply:
Container App and Cosmos Database are already included. Cosmos Database collection isn't a searchable Azure resource. I'll get Container App Environment added.